### DISCUSSION TITLE
局部导入SarPopout组件，怎么样式乱的啊？

### DISCUSSION THREAD

**Discussion by elarbor** [sutras/sard-uniapp#49] [General]
![image](https://github.com/user-attachments/assets/d2f64397-8eb0-4766-a232-b8de182d500b)
![image](https://github.com/user-attachments/assets/fb1d4509-e1bb-42e5-b6d1-651d728fdbb3)

![image](https://github.com/user-attachments/assets/73a1ff0b-0e40-4088-b2fe-da99030df6ea)

应该是我点摁扭的时候才出来SarPicker才对啊，直接就显示了？

**Comment by sutras** [comment]
v1.8.1已修复这个问题。
出现这个问题的原因是，编译script setup后的代码没有声明需要导入的组件，如果将模板也进行编译则不会有这个问题，但在uniapp上不允许编译模板；用easycom也不会出现识别不了组件的问题，因为uniapp做了处理。因此在编译时，我用正则识别依赖的组件，并将依赖插入到组件依赖声明上。
